### PR TITLE
:sweatsmile: fixed initialization in robotMap for moveController

### PR DESCRIPTION
**Pull request by doawelul**

### CODE REVIEW

**Comment by brettle** [comment]
@lhmcgann, if you are referring to issue #21, I created it as an issue so that it can be addressed separately from this pull request since it is independent of what this pull request is trying to fix.

File: Robot2018/src/org/usfirst/frc/team199/Robot2018/subsystems/Drivetrain.java [hunk 1]
```diff
@@ -51,7 +50,7 @@
 	private final DoubleSolenoid dtGear = RobotMap.dtGear;
 
 	public void pushGear(boolean forw) {
```
Comment (lhmcgann) [inline Robot2018/src/org/usfirst/frc/team199/Robot2018/subsystems/Drivetrain.java]: Change the name of this method to make more sense

File: Robot2018/src/org/usfirst/frc/team199/Robot2018/subsystems/Drivetrain.java [hunk 2]
```diff
@@ -134,46 +199,79 @@ public void initDefaultCommand() {
 		setDefaultCommand(new TeleopDrive());
```
Comment (lhmcgann) [inline Robot2018/src/org/usfirst/frc/team199/Robot2018/subsystems/Drivetrain.java]: can you move the default command to the top por favor so it's easy to find and not buried in here?

File: Robot2018/src/org/usfirst/frc/team199/Robot2018/subsystems/Drivetrain.java [hunk 3]
```diff
@@ -134,46 +199,79 @@ public void initDefaultCommand() {
...
 	public void enableTurnPid() {
 		turnController.enable();
 	}
 
+	/**
+	 * Sets the setPoint of the turnPID PIDController
+	 * 
+	 * @param set
+	 *            The value to set the setPoint at
+	 */
```
Comment (lhmcgann) [inline Robot2018/src/org/usfirst/frc/team199/Robot2018/subsystems/Drivetrain.java]: change method name por favor! like setTurnTarget. Same for setSetMove --> setDriveTarget (bc move is very general)

File: Robot2018/src/org/usfirst/frc/team199/Robot2018/subsystems/Drivetrain.java [hunk 4]
```diff
@@ -134,46 +199,79 @@ public void initDefaultCommand() {
...
+	public void disableMovePid() {
+		moveController.disable();
+	}
+
+	/**
+	 * Sets the setPoint of the movePID PIDController
+	 * 
+	 * @param set
+	 *            The value to set the setPoint at
+	 */
```
Comment (lhmcgann) [inline Robot2018/src/org/usfirst/frc/team199/Robot2018/subsystems/Drivetrain.java]: can the "move" PID in general be changed to "drive"? it's just a little clearer bc "move" can refer to turning to technically. I mean "drive" can as well but not as much

File: Robot2018/src/org/usfirst/frc/team199/Robot2018/subsystems/Drivetrain.java [hunk 5]
```diff
@@ -183,45 +281,45 @@ public double pidGet() {
 		return average(leftEnc.getDistance(), rightEnc.getDistance());
```
Comment (lhmcgann) [inline Robot2018/src/org/usfirst/frc/team199/Robot2018/subsystems/Drivetrain.java]: javadocs for this method???

File: Robot2018/src/org/usfirst/frc/team199/Robot2018/subsystems/Drivetrain.java [hunk 6]
```diff
@@ -183,45 +281,45 @@ public double pidGet() {
...
-	public void disableMovePid() {
-		moveController.disable();
-	}
-
-	public void setSetMove(double set) {
-		moveController.setSetpoint(set);
-	}
 
 	@Override
 	public void pidWrite(double output) {
```
Comment (lhmcgann) [inline Robot2018/src/org/usfirst/frc/team199/Robot2018/subsystems/Drivetrain.java]: javadocs?

**Comment by lhmcgann** [comment]
got it 👍

File: Robot2018/src/org/usfirst/frc/team199/Robot2018/subsystems/Drivetrain.java (hunk 5 above)
Comment (doawelul) [reply Robot2018/src/org/usfirst/frc/team199/Robot2018/subsystems/Drivetrain.java]: Implemented method for overriding PIDSource, I think it's unnatural to add javadocs for methods that aren't yours :)

File: Robot2018/src/org/usfirst/frc/team199/Robot2018/subsystems/Drivetrain.java (hunk 6 above)
Comment (doawelul) [reply Robot2018/src/org/usfirst/frc/team199/Robot2018/subsystems/Drivetrain.java]: Please check lastest commit.

**Pull request merged**